### ISSUE TITLE
IK IK IK

### ISSUE THREAD

**Issue by Sade39boi** [terraphantm/MS45-Flasher#2]
sorry not to sound rude or too proud but I have tried to use this for like a month and cant find any way to open the application I have ediabas installed i have ista+ and bmw tools i am very lost in using the program i cant even open it and really want to read my DME and then figure out rom raider rn I dont have any idea but slowly im getting into it i figure out ncs expert and the installation of ista and inpa and all its tool and idk i just want to make this work could you guide me into how I should open this application

**Comment by kmalinich** [comment]
What have you tried so far? I'm assuming you downloaded the release zip and extracted it, and opened the exe ..

**Comment by kmalinich** [comment]
> sorry not to sound rude or too proud but I have tried to use this for like a month and cant find any way to open the application I have ediabas installed i have ista+ and bmw tools i am very lost in using the program i cant even open it and really want to read my DME and then figure out rom raider rn I dont have any idea but slowly im getting into it i figure out ncs expert and the installation of ista and inpa and all its tool and idk i just want to make this work could you guide me into how I should open this application
> 

What have you tried so far? I'm assuming you downloaded the release zip and extracted it, and opened the exe ..

**Comment by Sade39boi** [comment]
yes I downloaded the master zip and there seems to be no exe file it's all just sln and cjprog files

**Comment by kmalinich** [comment]
> yes I downloaded the master zip and there seems to be no exe file it's all just sln and cjprog files

No. Download the release zip. It's on the Releases tab. You downloaded the source.

**Comment by Sade39boi** [comment]
oh okay so I downloaded wrong got you

**Comment by Sade39boi** [comment]
omg thank you I appreciate it so much i still am understanding github everyday its a crazy website with many buttons :'D

**Comment by Sade39boi** [comment]
got the application working thank you so much

**Comment by kmalinich** [comment]
@terraphantm you can probably close this
